main.js

```diff
@@ -0,0 +1,66 @@
+console.log('holla from js');
+
+var expenseName = document.forms[0].querySelector('input[name="expense-name"]');
+var expenseAmount = document.forms[0].querySelector('input[name="expense-amount"]');
+var expenseTableHead = document.querySelector('.expenses').children[0];
+var expenseTableBody = document.querySelector('.expenses').children[1];
+var inputForm = document.querySelector('.inputForm');
+var pressButton = document.querySelector('#add-expense');
+var totalExpense = document.querySelector('.expense-total');
+// var tableArr = [];
+
+var handleClick = function(evt) {
+  //gaurd against blank rows
+  if (expenseName.value === '' || expenseAmount.value === '') {
+    //broken error message next to button
+    // var error = document.createElement('p')
+    // error.textContent = 'Blank expenses, scrub?'
+    // inputForm.appendChild(error);
+    alert('Enter your expenses, scrub');
+  } else {
+  //create new table row
+    var newRow = document.createElement('tr');
+    newRow.innerHTML = '<td>' + expenseName.value + '</td><td>' + expenseAmount.value + '</td><button class="deleteButton" type="button">Delete</button></td>';
+    expenseTableBody.appendChild(newRow);
+    //push rows as objects to tableArr
+    tableArr = [];
+    for (var i = 0; i < expenseTableBody.rows.length; i++){
+      tableArr.push({
```

Comment:
It's cleaner to just use the `expenseName.value` and expenseAmount.value` that you already have above before you stick it in the DOM.